netbout-client/src/test/java/com/netbout/mock/MkInboxTest.java

```diff
@@ -46,12 +50,25 @@
      */
     @Test
     public final void testStart() throws IOException {
-        final String alias = "current-name";
-        final MkInbox inbox = new MkInbox(new H2Sql(), alias);
+        final String name = "current-name";
+        final Sql sql = new H2Sql();
+        final Aliases aliases = new MkUser(
```

Comment:
@pinaf because inside <code>MkInbox.start</code> it invites yourself as a friend. But self is not inserted as a alias yet, so invite will throw exception, <code>Friends.UnknownAliasException</code>, because of this reason